infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -196,11 +203,30 @@
                     + " --version                                                                "
                     + "              Show version\n";
 
+    private static volatile CorfuServer ACTIVE_SERVER;
+
+    private static volatile boolean SHUTDOWN_SERVER = false;
```

Comment:
I think you can make it lower case, since it's not a `static final`.
Same for `CLEANUP_SERVER`